tests/30rooms/60anonymousaccess.pl

```diff
@@ -1,5 +1,6 @@
-use Future::Utils qw( try_repeat_until_success );
+use Future::Utils qw( try_repeat_until_success repeat );
 use JSON qw( encode_json );
+use Data::Dumper;
```

Comment:
Delete this extraneous `use`.

Also, be aware that `Data::Dump`'s `pp` function is imported by `run-tests.pl`, so that's free to use anywhere in a test.